libcnb-common/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Could you turn this into a symlink, for parity with the other crates? :-)

Reply:
Ooops, sure. :)